cmd/provisioner-localpv/app/start.go

```diff
@@ -109,6 +111,12 @@ func Start(cmd *cobra.Command) error {
 		pvController.LeaderElection(isLeaderElectionEnabled()),
 	)
 	klog.V(4).Info("Provisioner started")
```

Comment:
Could you please move this log line, so that it is executed just before pc.Run(ctx) call is made?

Reply:
This order:
-- if block for analytics
-- log line
-- pc.Run(ctx) call